Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/FilterAdapter.cpp

```diff
@@ -116,15 +121,15 @@ namespace AZ::DocumentPropertyEditor
                         {
                             if (currMatch->m_matchesSelf)
                             {
-                                // node matches directly. All descendents automatically match if m_includeAllMatchDescendents is set
-                                if (!m_includeAllMatchDescendents)
+                                // node matches directly. All descendants automatically match if m_includeAllMatchDescendants is set
+                                if (!m_includeAllMatchDescendants)
                                 {
                                     cullUnmatchedChildRows(*valueIter, *matchIter);
                                 }
                             }
-                            else if (!currMatch->m_matchingDescendents.empty())
+                            else if (!currMatch->m_matchingDescendants.empty())
                             {
-                                // all nodes with matching descendents must be included so that there is a path to the matching node
+                                // all nodes with matching descendants must be included so that there is a path to the matching node
```

Comment:
To make it more confusing, I think the terms could probably be used interchangeably in some cases; although, it does change the meaning a bit. Descendants are descendent from their ancestors, or so I've read.

Reply:
Haha! goooood tiiiimes....